### ISSUE TITLE
Text alignment controls lack semantic indication of the active state

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#18721]
In WordPress 5.2, the text alignment buttons of the block toolbar were rendered as a group of buttons. The active state was communicated with an `aria-pressed` attribute. This made the visual feedback match the semantic feedback and assistive technology users were able to understand which alignment was set:

<img width="601" alt="Screenshot 2019-11-25 at 16 51 39" src="https://user-images.githubusercontent.com/1682452/69556022-5bf41f00-0fa4-11ea-8248-8b0d16b2f963.png">

In WordPress 5.3 (not sure since which Gutenberg version) the text alignment setting in the block toolbar was refactored to use a drop down menu. The buttons within the drop down don't have any kind of semantic indication of the active state. Assistive technology users (and any other software as well) now don't have any clue of what alignment is set. There's only a visual indication, no semantics at all:

<img width="882" alt="Screenshot 2019-11-25 at 16 49 07" src="https://user-images.githubusercontent.com/1682452/69556125-85ad4600-0fa4-11ea-9a27-1838d87d9b0f.png">

This is an accessibility regression, as the previous implementation had the proper semantics.

Worth also reminding that this drop down component renders an ARIA `menu` which can contain only three kind of elements:
- a generic `menuitem`
- a `menuitemcheckbox` for multiple choices
- a `menuitemradio` for single choices

In this case, since the text alignment can be set only to one value at a time, the menu items should use a `menuitemradio` role instead of a generic `menuitem`.

Seems to me the drop down menu items should use the `MenuItem` component instead of `IconButton` because `MenuItem` already has all that's needed and it's already used for this purpose, for example in the "More" menu in the Editor main toolbar.

Please consider to:
- use `MenuItem` for the items
- pass a `role="menuitemradio"` prop
- pass a `isSelected={ control.isActive }` prop which will render an `aria-checked` attribute with a true/false value depending on the item state

**Comment by afercia** [comment]
Please notice arrows navigation through the menu items should be preserved when changing the items to `MenuItem`:
- arrowing should loop through the items
- page scroll should be prevented
- writing flow should be prevented from kicking in
